pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -3269,3 +3273,13 @@ func mustNewPodInfo(t *testing.T, pod *v1.Pod) *framework.PodInfo {
 	}
 	return podInfo
 }
+
+// BuildNodeInfos build NodeInfo slice from a v1.Node slice
+func BuildNodeInfos(nodes []*v1.Node) []*framework.NodeInfo {
```

Comment:
We can **not** use `testing/framework.BuildNodeInfos` in package `framework/runtime`, because it will cause a cycle import. So I made a copy of the function here.